### PR TITLE
Add confirmation step for refund Approve/Deny buttons.

### PR DESCRIPTION
**Pull request by jimabramson**
XCOM-344

@rlucioni please review.  I'll pull in doc and accessibility once the general approach is validated.

### CODE REVIEW

**Comment by jimabramson** [comment]
re-pushed - previous push was missing some content.

**Comment by clintonb** [comment]
:+1: once tests are updated.

**Comment by rlucioni** [comment]
@jimabramson looks like the Selenium tests of the refund dashboard need to be updated to include coverage of the confirmation dialogue.

**Comment by jimabramson** [comment]
@rlucioni updated tests.

**Comment by jimabramson** [comment]
@rlucioni these tests consistently pass when i run them locally.  5 seconds seems like more than enough time to wait for the modal to appear - got any ideas?

File: ecommerce/extensions/dashboard/refunds/tests/test_acceptance.py [hunk 1]
```diff
@@ -64,7 +78,7 @@ def assert_alert_displayed(self, alert_class, text):
         self.assertIn(alert_class, classes)
         self.assertEqual(alert.find_element_by_css_selector('.message').text, text)
 
-    @skip("Requires refund processing endpoint.")
+    @skip("Requires refund processing endpoint.  Move to acceptance_tests and e-enable under ticket XCOM-342.")
```
Comment (rlucioni) [inline ecommerce/extensions/dashboard/refunds/tests/test_acceptance.py]: Not important, but I feel compelled to point it out: s/e-enable/re-enable

Comment (jimabramson) [inline ecommerce/extensions/dashboard/refunds/tests/test_acceptance.py]: it's for, you know, "e-commerce"

**Comment by jimabramson** [comment]
@rlucioni since the build is happy now, i've corrected the typo and squashed all the fixup commits

**Comment by jimabramson** [comment]
@griffresch  @srpearce  mind taking a quick peek to confirm these modal messages are acceptable?

initial state
screen shot 2015-05-28 at 2 06 13 pm

after clicking "approve"
screen shot 2015-05-28 at 2 05 34 pm

after clicking "deny"
screen shot 2015-05-28 at 2 05 50 pm

**Comment by srpearce** [comment]
@jimabramson I'd change "complete" to "full" in the first one; otherwise :+1:

**Comment by rlucioni** [comment]
@jimabramson :+1:

**Comment by jimabramson** [comment]
tweaked as requested by @srpearce

**Pull request merged**